adam/perception/high_level_semantics_situation_to_developmental_primitive_perception.py

```diff
@@ -1049,6 +1049,12 @@ def _instantiate_object_schema(
 
         # find axes for the object
         if situation_object:
+            # use the debug handle from the situation object if it is available, as it is more
+            # specific in the case of people (e.g. mom, dad, baby) than the debug handle
+            # generated from the ObjectStructuralSchema
+            debug_handle = (
+                situation_object.ontology_node.handle + "_" + debug_handle.split("_")[1]
```

Comment:
Some questions:
* What if the debug handle lacks a `_`?  
* If the ontology node handle and the debug handle are the same, do we want to duplicate? If this something which frequently happens?
* Does this make the debug logs any less readable?

Reply:
By the look of it (Running the m6 experiment script), the debug logs mostly refer to the spoken utterance (seemingly unaffected by this change), or else refer to specific subcomponents of an object (e.g. head_0 - for the head of a person).

prior to this change, the ontology node handle (of the schema) and debug handle appear to always be the same. After this change, the debug handle and ontology node handle (of the schema) only differ in the case of people.